settings/pyls-ms.vim

```diff
@@ -0,0 +1,32 @@
+augroup vimlsp_settings_pyls_ms
+  au!
+  LspRegisterServer {
+      \ 'name': 'pyls-ms',
+      \ 'cmd': {server_info->lsp_settings#get('pyls-ms', 'cmd', [lsp_settings#exec_path('pyls-ms')])},
+      \ 'root_uri':{server_info->lsp_settings#get('pyls-ms', 'root_uri', lsp_settings#root_uri(g:lsp_settings_root_markers))},
+	  \ 'initialization_options': lsp_settings#get('pyls-ms', 'initialization_options', {
+      \   'analysisUpdates': v:true,
+      \   'asyncStartup': v:true,
+      \   'displayOptions': {},
+      \   'interpreter': {
+      \      'properties': {
+	  \        'InterpreterPath': split(system('where python'), "\n")[0],
```

Comment:
No idea what `where` is, but it doesn't (normally) exist on macOS/Linux from what I can tell. Perhaps it's better to use `exepath()`?

```diff
-'InterpreterPath': split(system('where python'), "\n")[0],
+'InterpreterPath': exepath('python'),
```

Unsure how it works together with a virtualenv though.

Reply:
Fixed in master branch